config/openshift/kustomization.yaml

```diff
@@ -0,0 +1,43 @@
+# Adds namespace to all resources.
+namespace: openshift-kubevirt-ipam-controller-system
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: kubevirt-ipam-controller-
```

Comment:
I think this must match w/ the namespace name we have above (line 2).

Reply:
i would say dont do kustomize at all, provide static manifest imho